lib/src/blaze/block_witness_data.rs

```diff
@@ -457,25 +494,37 @@ impl BlockAndWitnessData {
         }
     }
 
-    pub async fn get_note_witness(
+    async fn get_note_witnesses<D, Spend, TreeGetter, OutputsFromTransaction, Node>(
```

Comment:
Is this name change correct?   `witness` to `witnesses`?

Reply:
Fixed